app/backend/wells/migrations/0106_auto_20200116_2327.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.2.9 on 2020-01-16 23:27
+
+from django.db import migrations
+
+
+INSERT_DRILLING_COMPANIES_INTO_ORG = """
+    INSERT INTO registries_organization (create_user, create_date, update_user, update_date, org_guid, name, effective_date, expiry_date, province_state_code)
+        SELECT create_user, create_date, update_user, update_date, drilling_company_guid, name, '2018-01-01 00:00:00', '9999-12-31 00:00:00', 'BC' FROM drilling_company;
+"""
+
+# Update any wells to use the same GUID which will point to the drilling companies inserted above
+UPDATE_WELL_ORG_TO_POINT_TO_OLD_DRILLING_COMPANY_GUIDS = """
+    UPDATE well
+    SET org_of_person_responsible_guid = drilling_company_guid
+    WHERE org_of_person_responsible_guid IS NULL;
+"""
+
+# Update any legacy submissions to use the same GUID which will point to the drilling companies inserted for the FK-ed well
+UPDATE_LEGACY_SUBMISSION_TO_OLD_DRILLING_COMPANY_GUID = """
+    UPDATE activity_submission s1
+    SET org_of_person_responsible_guid = drilling_company_guid
+    FROM activity_submission s2
+        INNER JOIN well w ON s2.well_tag_number = w.well_tag_number
+    WHERE
```

Comment:
Does this just ensure that only activity reports tied to a record in the well table are updated?

Reply:
Yes. If this SQL isn't run then the legacy records already created will still display (in the UI) nothing for the "company of the person responsible". Maybe I should get you some SQL to run against production to know how many of these legacy records exist, how many have a `org_of_person_responsible` is NOT NULL. Would that be helpful to know before this SQL is run?